configs/yolo/yolov3_d53_320_273e_coco_ipu.py

```diff
@@ -0,0 +1,98 @@
+_base_ = './yolov3_d53_mstrain-608_273e_coco.py'
+# dataset settings
+IM_SIZE = 320
+custom_hooks = [dict(type='NumClassCheckHook'), dict(type='BNToFP32')]
+img_norm_cfg = dict(mean=[0, 0, 0], std=[255., 255., 255.], to_rgb=False)
+train_pipeline = [
+    dict(type='LoadImageFromFile', channel_order='rgb'),
+    dict(type='LoadAnnotations', with_bbox=True),
+    dict(
+        type='Expand',
+        mean=img_norm_cfg['mean'],
+        to_rgb=img_norm_cfg['to_rgb'],
```

Comment:
Is there any inconsistency between `img_norm_cfg['to_rgb']` and `channel_order='rgb'`

Reply:
When the image is loaded, it is already rgb, so here is False